README.md

```diff
@@ -3,3 +3,5 @@ Share your git tricks here.
 
 ## Branches
 - `git push --delete  <branch_name>`: Delete remote branch.  see: http://stackoverflow.com/questions/2003505/delete-a-git-branch-both-locally-and-remotely
+
+- 'git branch -b <branch_name>': Create and checkout branch.
```

Comment:
**Correction:** `git checkout -b <new_branch>`: Simultaneously creates and checks out <new_branch>